_content/word-list.md

```diff
@@ -82,131 +87,171 @@ Virtual Host
 
 : Avatar-based entertaining live streaming.
 
-### Add-on, SDK and others
+### Other Agora-related names
 
+:::{admonition} Note
+When referring to a specific Agora SDK, use the definite article. See [](language-and-grammar.md#definite-articles-with-agora-sdks).
```

Comment:
See [Definite articles with Agora SDKs](language-and-grammar.md#definite-articles-with-agora-sdks)